src/components/Hero/HeroExpenses.jsx

```diff
@@ -114,23 +173,23 @@ function HeroExpenses() {
 
   return (
     <div>
-    <div className={css["hero-wrapper"]}>
-      <FormTransaction
-        selectOptions={selectOptionsExpenses}
-        onAddTransaction={handleAddTransaction}
-        type={type} 
-      />
-      <div className={css["hero-wrapper-tables"]}>
-        <TransactionTable
-          className={css["hero-transaction-table"]}
-          transactions={transactions}
+      <div className={css["hero-wrapper"]}>
+        <FormTransaction
+          selectOptions={selectOptionsExpenses}
+          onAddTransaction={handleAddTransaction}
           type={type}
-          handleDelete={handleDelete}
         />
-        {!isTablet && <SummaryTable data={data} />}
+        <div className={css["hero-wrapper-tables"]}>
+          <TransactionTable
+            className={css["hero-transaction-table"]}
+            transactions={transactions}
+            type={type}
+            handleDelete={handleDelete}
+          />
+          {!isTablet && <SummaryTable data={summaryData} />}
         </div>
       </div>
-      {isTablet && <SummaryTable data={data} />}
+      {isTablet && <SummaryTable data={summaryData} />}
     </div>
```

Comment:
to
data={summaryData} 
trzeba zamienić na 
summaryData={summaryData}
i strona będzie się wyświetlać